README.md

```diff
@@ -59,15 +59,15 @@ import { avm, pvm, evm } from '@avalabs/avalanchejs';
 ## Importing Essentials
 
 ```ts
-import { avm /** X-chain */, pvm /** P-chain */, evm /** C-chain */, utils } from "@avalabs/avalanchejs"
+import { avm /** X-chain */, pvm /** P-chain */, evm /** C-chain */, utils, secp256k1 } from "@avalabs/avalanchejs"
 
 // example calls
 const exportTx = avm.newExportTx(...) // constructs a new export tx from X
 const addValidatorTx = pvm.newAddPermissionlessValidatorTx(...) // constructs a new add validator tx on P
 const importTx = evm.newImportTx(...) // constructs a new import tx to C
 
 const publicKeyBytes = utils.hexToBuffer(publicKeyHex)
-const signature = utils.signHash(bytes, privateKeyBytes)
+const signature = secp256k1.signHash(bytes, privateKeyBytes)
```

Comment:
I think this is much clearer than `crypto.signHash` or `utils.signHash` since we're going to adding more crypto libs like BLS and Ed25519 in the future

Reply:
I agree it's clearer however the users have to know which crypto function to use when, which can be a bit confusing without deeper background knowledge. E.g: When to use `secp256k1.signHash` vs `bls.sign`.
This is something we can solve with better examples and documentation though.